### PR TITLE
[du] full sending on type annotations

### PR DESCRIPTION
**Pull request by tacastillo**

### CODE REVIEW

**Comment by tacastillo** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/21726?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#21726** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/21726?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#21725** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/21725?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tacastillo and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: docs/dagster-university/pages/dagster-essentials/lesson-3/defining-your-first-asset.md [hunk 1]
```diff
@@ -65,3 +65,5 @@ Your first asset, which you’ll name `taxi_trips_file`, will retrieve the yello
       ```
 
 That’s it - you’ve created your first Dagster asset! Using the `@asset` decorator, you can easily turn any existing Python function into a Dagster asset.
+
+Questions about the `-> None` bit? That's a Python feature called type annotations. In this case, it's saying that the function returns nothing. You can learn more about type annotations in the [Python documentation](https://docs.python.org/3/library/typing.html). We highly recommend using type annotations in your code to make it easier to read and understand.
```
Comment (erinkcochran87) [inline docs/dagster-university/pages/dagster-essentials/lesson-3/defining-your-first-asset.md:69]:
```suggestion
**Questions about the `-> None` bit?** That's a Python feature called **type annotation**. In this case, it's saying that the function returns nothing. You can learn more about type annotations in the [Python documentation](https://docs.python.org/3/library/typing.html). We highly recommend using type annotations in your code to make it easier to read and understand.
```

Some formatting suggestions and subject+verb agreement fix

**Review by erinkcochran87 — APPROVED** [review]
Left one comment but otherwise lgtm

**Comment by tacastillo** [comment]
### Merge activity

* **May 13, 1:34 PM EDT**: @tacastillo started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/21726).

**Pull request merged**